src/master-playlist-controller.js

```diff
@@ -893,14 +917,9 @@ export class MasterPlaylistController extends videojs.EventTarget {
     }
 
     // In flash playback, the segment loaders should be reset on every seek, even
-    // in buffer seeks
-    const isFlash =
-      (this.mode_ === 'flash') ||
-      (this.mode_ === 'auto' && !videojs.MediaSource.supportsNativeMediaSources());
```

Comment:
I snuck this change into this PR because the mode can only be `flash` or `html5`, so checking for the case of `auto` is unnecessary.